content/python/concepts/math-module/terms/math-isnan/math-isnan.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Math Isnan Method'
+Description: 'Math-isnan method returns True if x is a NaN (not a number) and False if x is a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Modules'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+**`Math-isnan`** method belongs to the group of Number-theoretic and representation functions in the [math-module](https://www.codecademy.com/resources/docs/python/math-module) and is used to handle simple (not complex) numbers. Except when explicitly noted otherwise, this function expects a float-type numeric value.
```

Comment:
```suggestion
In Python, the **`math.isnan()`** function returns `True` if the given value is NaN (not a number) and `False` if the value is a number. 
This method belongs to the group of number-theoretic and representation functions in the [math module](https://www.codecademy.com/resources/docs/python/math-module) and is used to handle simple (not complex) numbers. Except when explicitly noted otherwise, this function expects a float-type numeric value.
```

Reply:
This explanation of the function is written at the bottom, just brought it up for better structure. It can be removed from the bottom.